### PR TITLE
have `_example/main.go` work as expected

### PR DESCRIPTION
**Pull request by JCOR11599**
Fixes: https://github.com/dolthub/go-mysql-server/issues/911

### CODE REVIEW

File: _example/main.go [hunk 1]
```diff
@@ -46,7 +46,7 @@ func main() {
 
 	config := server.Config{
 		Protocol: "tcp",
-		Address:  "localhost:3306",
+		Address:  "localhost:3307",
```
Comment (fulghum) [inline _example/main.go:49]: Why the change to 3307?

Comment (JCOR11599) [reply _example/main.go:49]: Oops this was just for it to work locally (MySQL occupies 3306). I'll change it back.

File: sql/json.go [hunk 2]
```diff
@@ -78,10 +78,12 @@ func (t jsonType) SQL(dest []byte, v interface{}) (sqltypes.Value, error) {
 		return sqltypes.NULL, nil
 	}
 
-	js, ok := v.(JSONValue)
-	if !ok {
+	// Convert to jsonType
+	jsVal, err := t.Convert(v)
```
Comment (fulghum) [inline sql/json.go:82]: Change looks good. Anything we can add to `json_test.go` that would have caught this for us?

**Comment by fulghum** [comment]
Change looks good; just one question about whether we could add a test that would have caught this case for us before a customer had to tell us.

**Review by fulghum — APPROVED** [review]
Looks good! Thanks for fixing this gap!
